src/content/en/updates/2016/12/imagecapture.md

```diff
@@ -0,0 +1,327 @@
+margin: 0 0 20px 0;
+vertical-align: top;
+max-width: 100%;
+}
+
+</style>
+
+# Take Photos &amp; Control Camera Settings {: .page-title }
+
+{% include "web/_shared/contributors/samdutton.html" %}
+
+`ImageCapture` is an API for capturing still images and controlling camera settings.
+
+The API enables control over camera hardware features such as zoom, focus mode,
+contrast, ISO and white balance, as well as enabling access to the maximum resolution capabilities of the camera. Previous techniques for
+taking photos on the Web have used snapshots from a video stream, which are
+lower resolution than those available for still images.
+
+`ImageCapture` is available in Chrome 56 on desktop and Android as an
+[Origin Trial](https://github.com/jpchase/OriginTrials/blob/gh-pages/developer-guide.md),
+or in Chrome Canary on desktop and Android with **Experimental Web Platform**
+features enabled.
+
+The API has four methods:
+
+* `takePhoto()` returns a `Blob` which can be downloaded, stored by the browser
+or displayed in an `img` element.
+* `grabFrame()` returns an `ImageBitmap` object which can be post-processed — 
+for example, to selectively change color values.
+* `getPhotoCapabilities()` returns a `PhotoCapabilities` object that provides
+access to available camera options and their current values.
+* `setOptions()` is used to configure
+[camera settings](http://www.w3.org/TR/image-capture/#photosettings) such as
+zoom, white balance or focus mode.
+
+`ImageCapture` gets access to media streams from cameras via `getUserMedia()`:
+
+    navigator.mediaDevices.getUserMedia({video: true})
+      .then(gotMedia)
+      .catch(error => console.error('getUserMedia() error:', error));
+
+    function gotMedia(mediaStream){
+      const mediaStreamTrack = mediaStream.getVideoTracks()[0];
+      const imageCapture = new ImageCapture(mediaStreamTrack);
+      console.log(imageCapture);
+    }
+
+Note: To choose between different cameras, such as the front and back camera on
+a phone, get a list of available devices via the
+`MediaDevices.enumerateDevices()` method, then set `deviceId` in `getUserMedia()`
+constraints — as per the demo [here](https://webrtc.github.io/samples/src/content/devices/input-output/).
+
+You can use `takePhoto()` to get a still image and then set it as the `src` of
+an `<img>`:
+
+    imageCapture.takePhoto()
+      .then(blob => {
+        image.src = URL.createObjectURL(blob);
+      })
+      .catch(error => console.error('takePhoto() error:', error));
+
+Use `grabFrame()` to get data for a frame of video and then draw
+it on a `<canvas>`:
+
+    imageCapture.grabFrame()
+      .then(imageBitmap => {
+        canvas.width = imageBitmap.width;
+        canvas.height = imageBitmap.height;
+        canvas.getContext('2d').drawImage(imageBitmap, 0, 0);
+      })
+      .catch(error => console.error('grabFrame() error:', error));
+
+You can try this out in the demo below — if you're using a browser that supports `ImageCapture`:
```

Comment:
`whatwebcando.today` has this information (under "Advanced Camera Controls").

Also, I submitted a while ago, but hasn't showed up yet: http://caniuse.com/#search=imagecapture (https://github.com/Fyrd/caniuse/blob/master/features-json/imagecapture.json) (maybe you guys can chime in https://github.com/Fyrd/caniuse/issues/2887? )